site/create-image.md

```diff
@@ -53,11 +53,28 @@ Each section can contain one or more valid Dockerfile commands and would look li
 ```dockerfile
 [after-fmw-install]
 RUN rm /some/dir/unnecessary-file
+COPY --chown=oracle:oracle files/my_additional_file.txt /u01
 
 [final-build-commands]
 LABEL owner="middleware team"
 ```
 
+#### `--additionalBuildFiles`
+
+This option provides a way to supply additional files to the image build command.
+All provided files are copied directly under the `files` subfolder of the build context.  
+To get those files into the image, additional build commands must be provided using the `additionalBuildCommands` options.
+Access to these files using a build command such as COPY or ADD should use the original filename 
```

Comment:
command such as COPY or ADD -> command, such as `COPY` or `ADD`,  (use code font for commands)